### PR TITLE
SIMPLY-4066: Add DS Components

### PR DESCRIPTION
**Pull request by oliviawongnyc**
**Description**

- Adds Reservoir (Design System) components to the new Registry Admin Parcel app.
- Adds snapshot and unit tests for the top-level AdminPage, as well as the LibraryDetails component.

**Notes**

- Uses createRoot instead of ReactDOM.render (https://reactjs.org/blog/2022/03/08/react-18-upgrade-guide.html#updates-to-client-rendering-apis), though I still get an error while testing.
- Reservoir CSS file cannot be imported. Edwin is working on this. See [DSD-1001.](https://jira.nypl.org/browse/DSD-1001)

### CODE REVIEW

**Review by samanthaandrews — CHANGES_REQUESTED** [review]
Just a few small things & a couple questions. Whenever the CSS stuff is added go ahead and re-request review!

File: package.json [hunk 1]
```diff
@@ -56,6 +59,8 @@
     "lint-staged": "^12.4.1",
     "parcel": "^2.5.0",
     "prettier": "2.6.2",
+    "process": "^0.11.10",
```
[the remark this answers is not in the document]
Comment (oliviawongnyc) [reply package.json:62]: Yeah, I am curious about this too. I didn't install in manually. I even uninstalled it, and it returned.... Should I be worried? Tried uninstalling again, we'll see what happens.

File: src/components/LibraryDetails.tsx [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+const LibraryDetails = ({ library }: LibraryDetailsProps) => {
+  const {
+    uuid,
+    basic_info: basicInfo,
+    urls_and_contact: urlsAndContact,
+    areas,
+    stages,
+  } = library;
+  const [libraryStage, setLibraryStage] = useState<string>('');
+  const [registryStage, setRegistryStage] = useState<string>('');
```
[the remark this answers is not in the document]
Comment (oliviawongnyc) [reply src/components/LibraryDetails.tsx]: Done!

File: package.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (samanthaandrews) [reply package.json:63]: Got it, thanks for checking!

**Review by samanthaandrews — APPROVED** [review]

**Pull request merged**